src/module.c

```diff
@@ -5100,6 +5169,18 @@ const RedisModuleString *RM_GetKeyNameFromModuleKey(RedisModuleKey *key) {
     return key ? key->key : NULL;
 }
 
+/* Returns a dbid of the key from RedisModuleKey. */
+int RM_GetDbIdFromModuleKey(RedisModuleKey *key) {
+    return key ? key->db->id : -1;
+}
+
+/* Returns the dbid currently being processed.
+ * There is no guarantee that the key name is always available, so this may return -1.
```

Comment:
```suggestion
/* Returns the database id of the key currently being processed.
 * There is no guarantee that this info is always available, so this may return -1.
```